docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 sphinx==3.4.3
 -e git+https://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
 sphinxcontrib-katex
+sphinx-copybutton==0.4.0
```

Comment:
Since this is a small extension, do we need to pin the version number?

Reply:
At least is is guaranteed to work with pinned Sphinx version.

Reply:
In my opinion you can set it to >=0.9.0, pytorch sphinx theme works with it fine. (Tested it for torchvision)

Reply:
@oke-aditya well, copy-button latest is 0.4.0. >=0.9.0 is gallery you are talking about : https://github.com/pytorch/vision/blob/b7c235892332c49a86c4be1b00fb6a672a3f9930/docs/requirements.txt#L4

Reply:
My bad. Sorry for that